packages/h/hplip/files/hplip.metainfo.xml

```diff
@@ -1,15 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think you want to include this commit on this PR.

Reply:
I've removed it